### PR TITLE
bumpmonitoring package uses prcreator package for automatically creating PR

### PR DESCRIPTION
**Pull request by chaodaiG**

### CODE REVIEW

File: experiment/bumpmonitoring/main.go [hunk 1]
```diff
@@ -46,19 +51,92 @@ var (
...
+	var config string
+	var labelsOverride []string
+	var skipPullRequest bool
+
+	flag.StringSliceVar(&labelsOverride, "labels-override", nil, "Override labels to be added to PR.")
+	flag.BoolVar(&skipPullRequest, "skip-pullrequest", false, "")
+	flag.Parse()
+
+	var o options
+	data, err := ioutil.ReadFile(config)
```
Comment (cjwagner) [inline experiment/bumpmonitoring/main.go:69]: `config` is unset when used here so this will always fail.

File: experiment/bumpmonitoring/main.go [hunk 2]
```diff
@@ -46,19 +51,92 @@ var (
...
+	return nil
+}
+
+var _ bumper.PRHandler = (*client)(nil)
+
 type client struct {
-	srcPath string
-	dstPath string
+	SrcPath string
+	DstPath string
```
Comment (cjwagner) [inline experiment/bumpmonitoring/main.go:99]: nit: I don't think these need to be exported.

**Review by cjwagner — COMMENTED** [review]
Looks good except that the config file cannot be specified.

File: experiment/bumpmonitoring/main.go (hunk 2 above)
Comment (chaodaiG) [reply experiment/bumpmonitoring/main.go]: good point, done

File: experiment/bumpmonitoring/main.go (hunk 1 above)
Comment (chaodaiG) [reply experiment/bumpmonitoring/main.go:70]: nice catch, fixed

**Review by cjwagner — APPROVED** [review]

**Pull request merged**